#### Домашняя работа №4

##### Задание №1


1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. 
Найдите ее среднее значение и дисперсию.

In [1]:
import numpy as np

In [2]:
# M - среднее значение

M = (200 + 800) / 2
M

500.0

In [3]:
# D - дисперсия

D = ((800 - 200)**2) / 12
D

30000.0

Ответ: среднее значение М = 500.0; дисперсия D = 30000.0

##### Задание №2

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

In [4]:
# (x - 0.5)**2 / 12 = 0.2
# 12 * 0.2 = (x - 0.5)**2
# x - 0.5 = np.sqrt(12 * 0.2)

In [5]:
# x - правое значение величины В

x = np.sqrt(12 * 0.2) + 0.5
x

2.049193338482967

In [6]:
# M - среднее значение

M = (x - 0.5) / 2
M

0.7745966692414834

Ответ: правое значение величины В x = 2.049; среднее значение M = 0.774

##### Задание №3

3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения:

f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32).

Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

In [7]:
# M(X) = -2
# D(X) = 16
# std(X) = 4

Ответ: а) M(X) = -2; б) D(X) = 16; в) std(X) = 4

##### Задание №4

4) Рост взрослого населения города X имеет нормальное распределение. 
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
    
    а). больше 182 см
    б). больше 190 см
    в). от 166 см до 190 см
    г). от 166 см до 182 см
    д). от 158 см до 190 см
    е). не выше 150 см или не ниже 190 см
    ё). не выше 150 см или не ниже 198 см
    ж). ниже 166 см.

In [8]:
# a). больше 1 сигмы

P1 = 1 - 0.8413
P1

0.15869999999999995

In [9]:
# б). больше 2 сигм

P2 = 1 - 0.9772
P2

0.022800000000000042

In [10]:
# в). от -1 до 2 сигм

P3 = 1 - 0.0228 - 0.1587
P3

0.8185

In [11]:
# г). от -1 до 1 сигм

P4 = 1 - 0.1587 - 0.1587
P4

0.6825999999999999

In [12]:
# д). от -2 до 2 сигм

P5 = 1 - 0.0228 - 0.0228
P5

0.9543999999999999

In [13]:
# е). до -3 сигм + от 2 сигм

P6 = (1 - 0.9987) + (1 - 0.9772)
P6

0.02410000000000001

In [14]:
# ё). до -3 сигм + от 3 сигм

P7 = (1 - 0.9987) + (1 - 0.9987)
P7

0.0025999999999999357

In [15]:
# ж). менее 1 сигмы

P8 = 1 - 0.8413
P8

0.15869999999999995

Ответ: вероятность того, что случайным образом выбранный взрослый человек имеет рост:

    а). Р1 = 0.15869;
    б). Р2 = 0.02280;
    в). Р3 = 0.8185;
    г). Р4 = 0.68259;
    д). Р5 = 0.95439;
    е). Р6 = 0.02410;
    ё). Р7 = 0.00259
    ж). Р8 = 0.15869

##### Задание №5

5). На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от 
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [16]:
M = 178
sigma = 5

N_sig = (190 - M) / sigma
N_sig

2.4

Ответ: рост человека отклоняется от математического ожидания роста в популяции на 2,4 сигмы